### PR TITLE
[DOC] Simplify example for Ember.Route.modelFor

### PR DESCRIPTION
**Pull request by HeroicEric**
Some updates for the example for `Ember.Route.modelFor`

- Fixes indentation
- Removes unnecessary `resetNamespace`
- Updates example to avoid setting state on the route

### CODE REVIEW

File: packages/ember-routing/lib/system/route.js [hunk 1]
```diff
@@ -1855,9 +1855,9 @@ let Route = EmberObject.extend(ActionHandler, Evented, {
     // ...
 
     Router.map(function() {
-        this.route('post', { path: '/post/:post_id' }, function() {
-          this.route('comments', { resetNamespace: true });
-        });
+      this.route('post', { path: '/post/:post_id' }, function() {
+        this.route('comments', { resetNamespace: true });
```
Comment (locks) [inline packages/ember-routing/lib/system/route.js]: Sorry to drop this on you, but do you know if we need `resetNamespace` here?

Comment (HeroicEric) [inline packages/ember-routing/lib/system/route.js]: Hmm, that's a good point. That isn't needed to demonstrate `modelFor` and would probably be useful to make the example as simple as possible.

Now that I'm looking at the example, setting `post` as a prop on the route also seems strange (`this.set('post', this.modelFor('post'))`). Let me see if I can make the example a little bit more straightforward.

**Comment by HeroicEric** [comment]
@locks I updated the example to avoid setting state on the route and also removed the unneeded `resetNamespace` you mentioned.

**Comment by locks** [comment]
👌

**Pull request merged**